distribution/openhabhome/configurations/openhab_default.cfg

```diff
@@ -2618,3 +2618,17 @@ tcp:refreshinterval=250
 #
 # UPB Netword ID (required, must be a value between 0 and 255, inclusive)
 #upb:network=
+
+############################### PLC Logo Binding ####################################
+#
+# The refresh/polling the PLC interval default is 5S (5000mS). Tested down to 100mS
+# Value in mS
+#plclogo:refresh=5000
+#
+# Controller connection parameters, should match controller setup
+#plclogo:bcu.host=192.168.1.1
+#plclogo:bcu.remoteTSAP=0x2000
+#plclogo:bcu.localTSAP=0x3000
+#
+# Valid controller models are '0BA7' for LOGO7 and '0BA8' for LOGO8
+#plclogo:bcu.model=0BA8
```

Comment:
Please indicate in comment(s) which parameters are required vs. which are optional, and what the default value is if it's optional.